src/serializers/htmlToSlate/index.ts

```diff
@@ -109,13 +109,15 @@ const gatherTextMarkAttributes = ({ el, config = defaultConfig }: IgatherTextMar
         ...allAttrs,
         ...attrs,
       }
+      console.log([el, ...getChildren(el).flat()], allAttrs)
```

Comment:
For the added test (which picked up `em` and `u` for the `Our S` string), this console log gives:

```
[
      Element {
        parent: Element {
          parent: [Element],
          prev: null,
          next: null,
          startIndex: null,
          endIndex: null,
          children: [Array],
          name: 'em',
          attribs: {},
          type: 'tag'
        },
        prev: null,
        next: null,
        startIndex: null,
        endIndex: null,
        children: [ [Text] ],
        name: 'strong',
        attribs: {},
        type: 'tag'
      },
      Text {
        parent: Element {
          parent: [Element],
          prev: null,
          next: null,
          startIndex: null,
          endIndex: null,
          children: [Array],
          name: 'strong',
          attribs: {},
          type: 'tag'
        },
        prev: null,
        next: null,
        startIndex: null,
        endIndex: null,
        data: 'Our S',
        type: 'text'
      }
    ]
```

That would result in the serializer picking up on `bold` only.

Is it necessary to run `getChildren` multiple times? It's not the first time I have seen this type of behaviour. I have noted the `domparser2` libraries modify the DOM/objects as they go.

Reply:
This is expected. I misunderstood `getChildren`, which retrieves all children that are siblings.